packages/zeebe-element-templates-json-schema/src/defs/engines.json

```
@@ -0,0 +1,19 @@
+{
+  "$id": "#/engines",
+  "type": "object",
+  "description": "List of engines and their versions compatible with this version of the template.",
+  "default": {},
+  "examples": [
+    {
+      "camunda": "*"
```

Let's make this a proper "semantic version string", and not encourage wildcards as part of the example.

```suggestion
      "camunda": "^8.5"
```